### DISCUSSION TITLE
Welcome to ResourcefulBees Discussions!

### DISCUSSION THREAD

**Discussion by Epic428** [Resourceful-Bees/ResourcefulBees#85] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

^^^^^
Default template - too lazy to change

**Comment by Raff0rz** [comment]
Do you have a Discord link?

**Reply by Epic428** [reply]
https://discord.gg/ySYHEWQa

**Comment by Smazer87** [comment]
There is an issue with the Spooky Bee's not "pollinating" but are entering and leaving the hives. Is there a fix for this? Will an update be coming out to repair this glitch?

**Reply by Epic428** [reply]
If you’re playing on ATM6 the next pack update will have a fix. I believe they have a a PR on their github with all the changes to be made if you wanted to fix it before they update.
